src/node_watchdog.h

```diff
@@ -24,16 +24,24 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "uv.h"
-#include "node_mutex.h"
 #include <vector>
+#include "handle_wrap.h"
+#include "memory_tracker-inl.h"
```

Comment:
We generally shouldn’t be including `-inl.h` files from `.h` files – Can you move the `MemoryInfo()` method into the `.cc`file? That should remove the need for it.

Reply:
But I noticed that we usually include the "memory_tracker-inl.h" in most of the sources. 

I would like to open another PR for it, and following the conventions in this PR.